features/admin-query/src/main/feature/feature.xml

```diff
@@ -16,9 +16,10 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
-    <repository>mvn:ddf.features/admin/${ddf.version}/xml/features</repository>
-    <repository>mvn:ddf.features/security/${ddf.version}/xml/features</repository>
-    <repository>mvn:ddf.catalog/catalog-app/${ddf.version}/xml/features</repository>
+    <!-- Uncomment and put in version ranges once https://github.com/connexta/admin-console/issues/246 is complete. -->
```

Comment:
❓ How does the features still being used in the commented out feature files get found by this feature file?
❓ Should we add TODO to all these comments to make them easily searchable?

Reply:
I can add the TODO. The features files will need to be already included in the running Karaf instance in order for these features to be installed (this is the way the 1.2.x branch of `admin-console` works).